feed.go

```diff
@@ -37,6 +41,21 @@ type Item struct {
 	Content     string
 }
 
+// SerializeOpts represents serialization options for a feed.
+type SerializeOpts struct {
+	Indent int // how many spaces to indent
+}
+
+// get the indentation string for this option set
+func (s SerializeOpts) indentString() string {
+	return strings.Repeat(" ", s.Indent)
```

Comment:
I think you could build the string just once when the option is set. Some users may be building lot of feeds so this creates unnecessary garbage. SerializeOpts could just have an indent field of type `string`, and possibly a convenience method to set it to a number of spaces instead.